src/it/java/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/EmptyLineSeparatorTest.java

```diff
@@ -24,7 +24,7 @@ public static void setConfigurationBuilder() throws CheckstyleException, IOExcep
     @Test
     public void emptyLineSeparatorTest() throws IOException, Exception {
         
-        java.lang.Class<EmptyLineSeparatorCheck> clazz = EmptyLineSeparatorCheck.class;
+        Class<EmptyLineSeparatorCheck> clazz = EmptyLineSeparatorCheck.class;
```

Comment:
OK, I'll try to isolate the problem and I'll ask the question on
StackOverflow.
On 18 Aug 2015 22:30, "Roman Ivanov" <notifications@github.com> wrote:

> In
> src/it/java/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/EmptyLineSeparatorTest.java
> <https://github.com/checkstyle/checkstyle/pull/1750#discussion_r37349339>:
>
> > @@ -24,7 +24,7 @@ public static void setConfigurationBuilder() throws CheckstyleException, IOExcep
> >      @Test
> >      public void emptyLineSeparatorTest() throws IOException, Exception {
> >
> > -        java.lang.Class<EmptyLineSeparatorCheck> clazz = EmptyLineSeparatorCheck.class;
> > +        Class<EmptyLineSeparatorCheck> clazz = EmptyLineSeparatorCheck.class;
>
> the same compilation error for java8 and java7 , when I move that code to
> separate stanalone class to compile it by "javac" it works fine.
>
> +import java.lang.Class;
> +
>
>
> that is workaround for a problem, but importing "java.lang" most likely
> just another violation in Idea.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/checkstyle/checkstyle/pull/1750/files#r37349339>.
>